nexus/src/app/sagas/disk_create.rs

```diff
@@ -432,6 +457,21 @@ async fn sdc_regions_ensure(
     Ok(volume_data)
 }
 
+async fn sdc_regions_ensure_undo(
+    sagactx: NexusActionContext,
+) -> Result<(), anyhow::Error> {
+    let log = sagactx.user_data().log();
+    warn!(log, "regions_ensure_undo: Deleting crucible regions");
```

Comment:
what about the prefix? `sdc_regions_ensure_undo` instead?

Reply:
Done in 0d1f8bf4683fbc7e6561a6b3bc335a27df707432